From b9f982c6b7dcb2bfa6b4ee79aee48db298da2ae5 Mon Sep 17 00:00:00 2001
From: Changcheng Liu <changcheng.liu@intel.com>
Date: Thu, 31 Jan 2019 18:34:19 +0800
Subject: [PATCH 2/2] add osd__id paramter in ceph::osd puppet

If creating same osd_id number osd, remove the previous
one firstly.

Signed-off-by: Changcheng Liu <changcheng.liu@intel.com>

diff --git a/SPECS/puppet-ceph.spec b/SPECS/puppet-ceph.spec
index e5fca31..5b4d26e 100644
--- a/SPECS/puppet-ceph.spec
+++ b/SPECS/puppet-ceph.spec
@@ -18,6 +18,7 @@ Patch0005:      0005-Remove-puppetlabs-apt-as-ceph-requirement.patch
 Patch0006:      0006-ceph-disk-prepare-invalid-data-disk-value.patch
 Patch0007:      0007-Add-StarlingX-specific-restart-command-for-Ceph-moni.patch
 Patch0008:      0001-create-filestore-OSD-in-ceph-v13.2.2.patch
+Patch0009:      0002-add-osd_id-to-remove-osd.-osd_id-after-config-ceph.c.patch
 
 BuildArch:      noarch
 
@@ -41,6 +42,7 @@ Community Developed Ceph Module
 %patch0006 -p1
 %patch0007 -p1
 %patch0008 -p1
+%patch0009 -p1
 
 find . -type f -name ".*" -exec rm {} +
 find . -size 0 -exec rm {} +
-- 
2.17.1

